### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by umangahuja1** [sipah00/CrickFev#11]
I think it would be good to shift to Python3 for this as well as coming projects. What's your take?

**Comment by sipah00** [comment]
@umangahuja1, ya, it will be good to shift to py3.
There will be some minor changes.

**Comment by umangahuja1** [comment]
Shall I go ahead and do the changes?

**Comment by sipah00** [comment]
@umangahuja1, yes and after this, make sure that every command is working fine.

**Comment by kiryteo** [comment]
@umangahuja1 are you still working on this.?
